packages/kit/src/exports/index.js

```diff
@@ -2,12 +2,27 @@ import { HttpError, Redirect, ActionFailure } from '../runtime/control.js';
 import { BROWSER, DEV } from 'esm-env';
 import { get_route_segments } from '../utils/routing.js';
 
-// For some reason we need to type the params as well here,
-// JSdoc doesn't seem to like @type with function overloads
 /**
- * @type {import('@sveltejs/kit').error}
+ * @overload
  * @param {number} status
- * @param {any} message
+ * @param {App.Error} body
+ * @return {HttpError}
+ */
+
+/**
+ * @overload
+ * @param {number} status
+ * @param {{ message: string } extends App.Error ? App.Error | string | undefined : never} [body]
```

Comment:
These overloads are for the error function below right? The second param name should be message to match the function.